README.md

```diff
@@ -152,6 +155,39 @@ You supply either an IAM service **API key** or an **access token**:
 - Use the API key to have the SDK manage the lifecycle of the access token. The SDK requests an access token, ensures that the access token is valid, and refreshes it if necessary.
 - Use the access token if you want to manage the lifecycle yourself. For details, see [Authenticating with IAM tokens](https://cloud.ibm.com/docs/services/watson/getting-started-iam.html). If you want to switch to API key, override your stored IAM credentials with an IAM API key.
 
+##### ICP
+
+Like IAM, you can pass in credentials to let the SDK manage an access token for you or directly supply an access token to do it yourself.
+
+If letting the SDK manage the token, you must set `authentication_type` to `icp4d`.
```

Comment:
If you choose to let the SDK manage the token, `authentication_type` must be set to `icp4d`.